.travis.yml

```diff
@@ -1,3 +1,10 @@
 language: ruby
+
+sudo: false
```

Comment:
this is not recommended
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration